src/handler-builder.ts

```diff
@@ -140,6 +182,36 @@ export class HandlerBuilder {
     };
   }
 
+  /**
+   * Create a handler for Crashlytics events.
+  
+   * `issue.onNew` handles events where the app experiences an issue for the first time.
+
+   * @example
+   * ```javascript
+   * exports.myFunction = functions.handler.crashlytics.issue().onNew(async
```

Comment:
This should actually be `exports.myFunction = functions.handler.crashlytics.issue.onNew(async`. No brackets after issue. Everything in the handler namespace is formatted like this, no parameters. Same applies for the other examples for crashlytics.

Reply:
Got it -- removed the parens.  Hope these are correct now, thanks!